app/assets/javascripts/discourse/app/controllers/topic.js

```diff
@@ -364,6 +364,10 @@ export default class TopicController extends Controller {
     event?.preventDefault();
     if (this.get("model.details.can_edit")) {
       this.set("editingTopic", true);
+
+      schedule("afterRender", () => {
+        document.querySelector("#edit-title")?.focus();
+      });
```

Comment:
Can we put the `autoFocus` modifier on the input instead? This manual Ember runloop stuff is always tedious to debug in future.

Reply:
ah yes I can do this